packages/builder/src/components/common/RoleSelect.svelte

```diff
@@ -58,25 +88,51 @@
   const onChange = e => {
     if (allowRemove && e.detail === RemoveID) {
       dispatch("remove")
+    } else if (e.detail === CreatorID) {
+      dispatch("addcreator")
```

Comment:
NAB: Is there a special reason that choosing creator fires a unique event? Just listening to the normal `change` event and then checking the payload is creator would achieve the same thing wouldn't it? Just thinking along the lines of trying to keep this simple.